src/Fixer/Import/OrderedImportsFixer.php

```diff
@@ -68,10 +68,12 @@
     /**
      * Configuration settings for sort algorithm and sor types.
      *
+     * Use 'null' for sorting alphabetically by default.
```

Comment:
regardless there will be that comment or not - this behaviour is described in interface, so all classes that implements it must follow that.

currently: 

```php
$f = new OrderedImportsFixer(); // configuration is default
$f->configure(null); // configuration is still default
$f->configure($customArrayConfig); // configuration is custom
$f->configure(null); // configuration is still custom, while null should reset it do default one
```